### ISSUE TITLE
Wrapped not accurate

### ISSUE THREAD

**Issue by byruzyayandy1** [ishandeveloper/github-wrapped#23]
Wrapped not accurate, it reduces a lot of people's commits and PRs.
Fix the bug, Please!

**Comment by aaditkamat** [comment]
I have to agree, the ones shown on [GitHub README stats](github-readme-stats.vercel.app) are more accurate.

**Comment by ishandeveloper** [comment]
Hey @byruzyayandy1 Thank you for taking the time to create this issue. Actually, currently the stats are filtered to only include the contributions which were made in the past year (2020), any contributions made before Jan 1, 2020 are not counted.

Also, as it's mentioned on the website "contributions made towards the open-source community". Therefore, only the contributions made to Public Repositories are included for now.

However, We could certainly add an option to count private contributions as well, for next year's wrap : )

Can you please verify the stats on your github profile with the things I mentioned? Also, I hope this answers your doubt as well @aaditkamat : )

I'm keeping this issue open for now, in case you find anything. 

Happy New Year!

**Comment by aaditkamat** [comment]
> Also, as it's mentioned on the website "contributions made towards the open-source community". Therefore, only the contributions made to Public Repositories are included for now.

@ishandeveloper, it seems like the confusion has arisen because the contributions considered are commits made on public repositories owned by other users. Was that the intention of the application?

**Comment by byruzyayandy1** [comment]
@ishandeveloper, I think what @aaditkamat said is right, it seems like the confusion has arisen because the contributions considered are commits made on public repositories owned by other users. Was that initially the intention of the application?
